opencti-platform/opencti-front/src/private/components/common/form/AuthorizedMembersField.tsx

```diff
@@ -169,6 +183,16 @@ const AuthorizedMembersField = ({
           accessRight: 'admin',
         },
       ]);
+    } else if (showCurrentUserLine) {
```

Comment:
something is wrong with this condition, because it means we can't display creator & current user at the same time. Not sure about the condition on owner.